Classes/Helper/InlineHelper.php

```diff
@@ -26,6 +26,8 @@
  *  This copyright notice MUST APPEAR in all copies of the script!
  * ************************************************************* */
 
+use TYPO3\CMS\Backend\Utility\BackendUtility;
```

Comment:
please remove all changes made based on our own coding guideline
- we should leave the code style as it is and only apply the changes needed to fix the functionality